tests/console/validate_raid_encrypt_home.pm

```diff
@@ -0,0 +1,24 @@
+# SUSE's openQA tests
+#
+# Copyright 2024 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Simple RAID partitioning layout validation for encrypted home
+# Maintainer: QE YaST and Migration (QE Yam) <qe-yam at suse de>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+
+sub run {
+    select_console 'root-console';
+    record_info("lsblk -M");
+    assert_script_run "lsblk -M";
```

Comment:
we need to ensure that the crypt /home partition is under md1, I don't think you can ensure that with that grep, right? (if the bug would happen again this validation might not catch it...) Also you repeat the command to list and visualize stuff and that is not what it is a validation.

`lsblk` have some options to get the data in a different format and then you can iterate over that checking some keys for example. Perhaps there is some other command more direct.

Reply:
`tests/console/validate_blockdevices.pm` is able to validate disks, but I doubt it will work for raid, never was the intention. It is using lsblk -J to get the json and iterate, we basically just need to process the block:
```
md1
|
---cr_home
```
without regex, of course.

Reply:
but most likely you will have to iterate `lsblk -M -J /dev/md1` with `Mojo::JSON 'decode_json';`

Reply:
Updated code and VR.